src/interpreter/InterpretConst.ts

```diff
@@ -0,0 +1,25 @@
+
+import { CommonModel } from '../models/CommonModel';
+import { Schema } from '../models/Schema';
+import { inferTypeFromValue } from './Utils';
+
+/**
+ * Interpreter function for JSON Schema draft 7 const keyword.
+ * 
+ * @param schema 
+ * @param model
+ */
+export default function interpretConst(schema: Schema | boolean, model: CommonModel) {
+  if (typeof schema === 'boolean' || schema.const === undefined) return;
+  
+  const schemaConst = schema.const;
+  model.enum = [schemaConst];
+
+  //If schema does not contain type interpret the schema
+  if (schema.type === undefined) {
+    const inferredType = inferTypeFromValue(schemaConst);
+    if (inferredType !== undefined) {
+      model.setType(inferredType);
+    }
+  }
```

Comment:
Hmm... If someone pass `const` then it should (probably) override types. Am I wrong?

Reply:
With this approach, we favor already sat `type` keyword intstead of interpreting `const` as type. Which is why` if (schema.type === undefined) {` is there. This is just to keep it as we have it now with the old simplifier.

Reply:
We can change this behavior based on the interpretation options later, if we want to.

Reply:
Yep, but if you have `{ type: 'boolean', const: "foo" }` then you have invalid schema... however I agree to go with what is in the scheme.

Reply:
yep it would, but neither `type: 'boolean'` or `type: 'string'` is correct in this case 😄 

This is also one of the reasons why I suggested #148.

Reply:
Probably `const` doesn't combine with other types and only enforces one type. Maybe I don't understand JSON Schema correctly.

Reply:
If you try and validate `{ type: 'boolean', const: "foo" }` with some input data, you won't be able to because the `type` and `const` are contradicting. 

i.e. if your input is `"foo"` it is accepted by `const: "foo"` but is rejected by `type: 'boolean'`, and vice versa if the input is a boolean 😄 

So we cant really win if you create invalid schemas 😄